automatic/apache-httpd/update.ps1

```diff
@@ -0,0 +1,37 @@
+Import-Module au
+Import-module "$PSScriptRoot\..\..\scripts\au_extensions.psm1"
+
+function global:au_AfterUpdate { Set-DescriptionFromReadme -SkipFirst 1 }
+
+function global:au_GetLatest {
+    $downloadPageUrl = 'http://www.apachehaus.com/cgi-bin/download.plx'
+    $versionRegEx = 'httpd\-([0-9\.]+)\-x\d+\-(.*)\.zip'
+    $url32 = 'https://www.apachehaus.com/downloads/httpd-$version-x86-$vcNumber.zip'
```

Comment:
This doesn't need to be changed, it's not being used before it's defined. The string gets expanded after the vsNumber variable has a value.